troposphere/static/js/modals/instance/resume.js

```diff
@@ -1,16 +1,31 @@
 import ModalHelpers from "components/modals/ModalHelpers";
-
 import InstanceResumeModal from "components/modals/instance/InstanceResumeModal";
+
 import actions from "actions";
 
 
 export default {
+    /**
+     * Resume any valid/actionable resources
+     *
+     * The action method handles resources verifying type (via instanceof)
+     * to ensure that multi0ple resources can be handled.
+     *
+     * @param resources - collection of resources to act on
+     */
+    resume: function(resources) {
```

Comment:
To give some background in my thinking ... the argument of `modals.InstanceModals.resume` could be an instance or collection of instances. Consider that there is a feature flag. Part of the rationale for that is, "what is a given cloud provider is having issues with _'batch'_ `resume` actions and we need to disallow that quickly?" 

So, I was hoping to add some complexity by allowing the argument to be either of those (though, I will agree that we could filter out non-`Instance` models soon). 

I was aiming for an evaluation branch that was _workable_ but not perfect. We will won't know if "batch" actions will cause issues until we test that on a provider - and this branch's purpose to get to that point.